docs/source/index.rst

```diff
@@ -36,3 +36,15 @@ Indices and Tables
 .. _HRTF: https://en.wikipedia.org/wiki/Head-related_transfer_function
 .. _alure: https://github.com/kcat/alure
 .. _Cython: https://cython.org
+
+
+.. toctree::
+   :caption: Quick Navigation
+   :maxdepth: 1
```

Comment:
Thanks, TIL about multiple ToCs.  The maxdepth is unnecessary though and please move this above *Indices and Tables*, and caption the main ToC as *Table of Contents*.